src/huggingface_hub/hf_api.py

```diff
@@ -1911,7 +1911,15 @@ def list_models(
         if len(search_list) > 0:
             params["search"] = search_list
         if sort is not None:
-            params["sort"] = "lastModified" if sort == "last_modified" else sort
+            params["sort"] = (
+                "lastModified"
+                if sort == "last_modified"
+                else "trendingScore"
+                if sort == "trending_score"
+                else "createdAt"
+                if sort == "created_at"
+                else sort
+            )
```

Comment:
if it ever becomes a problem again in the future we might explore having a "snake_case to camelCase" helper to fix it once and for all. Ok to keep it like this for now (at least things are clearly documented and tested).

Reply:
beware, we like inconsistency in the hub team, we might use a snake case just to trick you! 🤪